providers/class-two-factor-email.php

```diff
@@ -142,14 +144,25 @@ public function user_token_lifetime( $user_id ) {
 	public function user_token_ttl( $user_id ) {
 		$token_ttl = 15 * MINUTE_IN_SECONDS;
 
+		/**
+		 * Number of seconds the token is considered valid
+		 * after the generation.
+		 *
+		 * @deprecated 0.11.0 Use {@see 'two_factor_email_token_ttl'} instead.
+		 *
+		 * @param integer $token_ttl Token time-to-live in seconds.
+		 * @param integer $user_id User ID.
+		 */
+		$token_ttl = (int) apply_filters_deprecated( 'two_factor_token_ttl', array( $token_ttl, $user_id ), '0.7.0', 'two_factor_email_token_ttl' );
```

Comment:
Decided to rename this to `two_factor_email_token_ttl` for clarity and consistency.